src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -132,6 +132,8 @@ struct rocblas_gemm
     void finalize(context& ctx, const shape& output_shape, const std::vector<shape>& input_shapes)
     {
 #ifdef MIGRAPHX_USE_ROCBLAS_TUNING_API
+        if(solution_idx == 0)
+            solution_idx = gemm_default_solution(ctx, output_shape, input_shapes);
```

Comment:
From what i understand, This is where problem cache would be helpful. So that it doesn't tune same gemm configs again and again and instead would return saved solution idx from previous tuning ? Is that correct ? 

But then,

Reply:
Yes thats correct.